diff --git a/vendor/fuego/smartgame/SgHash.h b/vendor/fuego/smartgame/SgHash.h
index 3aab6c0..06b925d 100644
--- a/vendor/fuego/smartgame/SgHash.h
+++ b/vendor/fuego/smartgame/SgHash.h
@@ -271,7 +271,7 @@ std::ostream& operator<<(std::ostream& out, const SgHash<N>& hash)
 }
 
 template<int N>
-std::ostream& operator>>(std::istream& in, const SgHash<N>& hash)
+std::istream& operator>>(std::istream& in, const SgHash<N>& hash)
 {
     std::string str;
     in >> str;
diff --git a/vendor/fuego/smartgame/SgPointSetUtil.h b/vendor/fuego/smartgame/SgPointSetUtil.h
index a9432c6..0c91077 100644
--- a/vendor/fuego/smartgame/SgPointSetUtil.h
+++ b/vendor/fuego/smartgame/SgPointSetUtil.h
@@ -64,7 +64,7 @@ public:
     std::istream& Read(std::istream& in) const;
 
 private:
-    mutable SgPointSet& m_pointSet; // allow temp objects to modify
+    SgPointSet& m_pointSet; // allow temp objects to modify
 };
 
 /** @relatesalso SgReadPointSet */
