diff --git a/vendor/fuego/go/GoGtpEngine.cpp b/vendor/fuego/go/GoGtpEngine.cpp
index 292cbbc..7ecd638 100644
--- a/vendor/fuego/go/GoGtpEngine.cpp
+++ b/vendor/fuego/go/GoGtpEngine.cpp
@@ -1608,7 +1608,7 @@ void GoGtpEngine::Ponder()
     // Call GoPlayer::Ponder() after 0.2 seconds delay to avoid calls in very
     // short intervals between received commands
     boost::xtime time;
-    boost::xtime_get(&time, boost::TIME_UTC);
+    boost::xtime_get(&time, boost::TIME_UTC_);
     bool aborted = false;
     for (int i = 0; i < 200; ++i)
     {
diff --git a/vendor/fuego/gtpengine/GtpEngine.cpp b/vendor/fuego/gtpengine/GtpEngine.cpp
index 4c4e56c..46b2808 100644
--- a/vendor/fuego/gtpengine/GtpEngine.cpp
+++ b/vendor/fuego/gtpengine/GtpEngine.cpp
@@ -349,7 +349,7 @@ void ReadThread::Function::ExecuteSleepLine(const string& line)
     {
         cerr << "GtpEngine: sleep " << seconds << '\n';
         xtime time;
-        xtime_get(&time, boost::TIME_UTC);
+        xtime_get(&time, boost::TIME_UTC_);
         time.sec += seconds;
         thread::sleep(time);
         cerr << "GtpEngine: sleep done\n";
