diff --git a/vendor/TouchXML/Common/Source/CXMLDocument.h b/vendor/TouchXML/Common/Source/CXMLDocument.h
index 369786c..44a75de 100644
--- a/vendor/TouchXML/Common/Source/CXMLDocument.h
+++ b/vendor/TouchXML/Common/Source/CXMLDocument.h
@@ -30,8 +30,8 @@
 #import "CXMLNode.h"
 
 enum {
-	CXMLDocumentTidyHTML, // Based on NSXMLDocumentTidyHTML
-	CXMLDocumentTidyXML, // Based on NSXMLDocumentTidyXML
+	CXMLDocumentTidyHTML = 1, // Based on NSXMLDocumentTidyHTML
+	CXMLDocumentTidyXML = 2, // Based on NSXMLDocumentTidyXML
 };
 
 @class CXMLElement;
diff --git a/vendor/TouchXML/Common/Source/CXMLDocument.m b/vendor/TouchXML/Common/Source/CXMLDocument.m
index 8f6a611..a157ea7 100644
--- a/vendor/TouchXML/Common/Source/CXMLDocument.m
+++ b/vendor/TouchXML/Common/Source/CXMLDocument.m
@@ -50,6 +50,7 @@ if ((self = [super init]) != NULL)
 	if (inOptions & CXMLDocumentTidyHTML)
 		{
 		inString = [[CTidy tidy] tidyString:inString inputFormat:TidyFormat_HTML outputFormat:TidyFormat_XHTML diagnostics:NULL error:&theError];
+		inString = [inString stringByReplacingOccurrencesOfString:@"xmlns=\"http://www.w3.org/1999/xhtml\"" withString:@""]; // remove default namespace... ugh
 		}
 	else if (inOptions & CXMLDocumentTidyXML)
 		{
